### DISCUSSION TITLE
Define script in compile firmware

### DISCUSSION THREAD

**Discussion by cepela** [arendst/Tasmota#13287] [Q&A] [answered]
Hello, 
it's posible define script when compile own firmware?

**Comment by barbudor** [comment, marked as answer]
Not as of today
This is already discussed here : https://github.com/arendst/Tasmota/discussions/13149
You didn't searched well

**Comment by cepela** [comment]
Thank you.

**Reply by gemu2015** [reply]
its already merged into development:  #13153
